sway/sway.5.scd

```diff
@@ -341,6 +341,30 @@ runtime.
 	*bindcode* [--whole-window] [--border] [--exclude-titlebar] [--release] [--locked] [--input-device=<device>] [--no-warn] <code> <command>
 	is also available for binding with key/button codes instead of key/button names.
 
+*bindswitch* [--locked] [--no-warn] <switch>:<state> <command>
+	Binds <switch> to execute the sway command _command_ on state changes.
```

Comment:
This doesn't need to expose implementation details such as `WLR_SWITCH_*`

Reply:
Fair, but it may be non-obvious what the switches refer to otherwise. I guess I did mention `...laptop lid and tablet mode switches...`